calculateMetrics/old_metrics.py

```diff
@@ -0,0 +1,319 @@
+"""Determine how the well the team worked together"""
```

Comment:
Is this file required? As it seems that the above file `metrics.py` does these calculations and more. If it is not necessary, I would suggest it being deleted.